tests/App/Entity/Foo.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sonata\MediaBundle\Model\MediaInterface;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="media__foo")
+ */
+class Foo
```

Comment:
This is how I have my entity in my project and it fails when deleting, but I can't make it fail on tests. Not sure how to configure doctrine to use proxies, it should be enabled by default right? but on tests I always receive the real object, not the proxy. (I tried clearing the entity manager or removing the object from the unit of work, but nothing works.

Reply:
Maybe @greg0ire can help